code/game/turfs/simulated/footsteps.dm

```diff
@@ -158,4 +158,16 @@
 			volume += shoes.step_volume
 			range += shoes.step_range
 
-		playsound(T, footsound, volume, 1, range)
\ No newline at end of file
+		playsound(T, footsound, volume, 1, range)
+
+/mob/living/carbon/human/proc/has_footsteps()
+	if(species.silent_steps || buckled || lying || throwing)
+		return //people flying, lying down or sitting do not step
+
+	if(shoes && (shoes.item_flags & ITEM_FLAG_SILENT))
+		return // quiet shoes
+
+	if(!has_organ(BP_L_FOOT) && !has_organ(BP_R_FOOT))
```

Comment:
Remove this check completely, necromorphs don't have feet

Reply:
> Remove this check completely, necromorphs don't have feet

Add one check if it is no necro

Reply:
no, just remove the check. Even if a human has their feet cut off, their leg stumps would still make a sound when walking on them

Reply:
> no, just remove the check. Even if a human has their feet cut off, their leg stumps would still make a sound when walking on them


The paradox is that here a person steps with his feet and this is clearly audible in the sound. It will be strange that a person without legs will walk and the sounds will be the same.

Reply:
that's a minor edge case which isn't important